content/docs/details/techdocs/index.md

```diff
@@ -0,0 +1,248 @@
+---
+title: Using Tech Docs in Roadie
+publishedDate: '2023-08-30T21:00:00.0Z'
+description: Using Techdocs in Roadie Backstage.
+---
+
+The TechDocs feature of Roadie Backstage allows markdown files written alongside the code of your components to appear in Backstage as styled HTML documentation. 
+
+![a page of basic documentation for a service in the Backstage service catalog](./sample-service-docs-backstage.png)
+
+
+## MkDocs Plugins
+
+Under the hood, Backstage uses the popular [MkDocs library](https://www.mkdocs.org/) to render documentation. This library is very feature rich thanks to its extensive range of available plugins.
+
+We currently include the following MkDocs plugins in our build process:
```

Comment:
Half of the description are in instruction format, half are describing the functionality of the plugin. Let's make those use the same grammar

Reply:
Not sure I follow why that's an issue - some of the sections are guides, some are information. What would you like to change this section to say for instance?

Reply:
I mean in the listing of plugins. The narrator differs between elements from passive to 'you':
* Lists things
* Do things
* Allows you to do things
* Do this thing

For readability it's better to keep it consistent

Reply:
Any better?

Reply:
Looks good :+1: